src/server/player_sao.h

```diff
@@ -182,6 +183,7 @@ class PlayerSAO : public UnitSAO
 	std::string generateUpdatePhysicsOverrideCommand() const;
 
 	RemotePlayer *m_player = nullptr;
+	std::string m_player_name; ///< used as GUID
```

Comment:
in theory there is no need to duplicate this data

Reply:
@sfence

Reply:
This should be resolved now. Comment added.